POM/test_5AE.py

```diff
@@ -0,0 +1,38 @@
+from base_pages import AutomationExercise, AELoginSignup
+from locators import AutomationExerciseLocators, AELSLocators
+from selenium.webdriver.common.by import By
+import os
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+
+'''
+1. Launch browser
+2. Navigate to url 'http://automationexercise.com'
+3. Verify that home page is visible successfully
+4. Click on 'Signup / Login' button
+5. Verify 'New User Signup!' is visible
+6. Enter name and already registered email address
+7. Click 'Signup' button
+8. Verify error 'Email Address already exist!' is visible
+'''
+
+
+def test_already_exist(get_driver):
+    email = os.environ.get("email")
+    name = os.environ.get("name")
+
+    driver = get_driver
+    driver.get(AutomationExercise.base_url)
+
+    assert driver.find_element(By.XPATH, AutomationExerciseLocators.home_icon)
```

Comment:
meaningful assert message might be helpful for you or a team member.

Reply:
Added